### PR TITLE
Update instructions

### PR DESCRIPTION
**Pull request by yuya-yuliya**

### CODE REVIEW

**Review by evgenii-nikulin — APPROVED** [review]

**Review by stanislav-kozorez — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -115,43 +114,33 @@ To bootstrap the Commerce Server follow these instructions:
...
 * Make sure the following exists in your web.config and if not then add it (system.webServer/handlers section):
 `<add verb="*" path="sitecorejss_media.ashx" type="Sitecore.JavaScriptServices.Media.MediaRequestHandler, Sitecore.JavaScriptServices.Media" name="Sitecore.JavaScriptServices.Media.MediaRequestHandler" />`
 
 ----
 ### Building & deploying Headless Commerce Accelerator      
-1. Fetch **Headless Commerce Accelerator** code base
+1. Fetch **Headless Commerce Accelerator** code base.
 2. Copy Sitecore license to **./src** folder.
-3. Local automation is implemented on the top of the [Cake tool](https://cakebuild.net/). Check **build/build.cake** if `Sitecore/Parameters.InitParams` are correct for your installation.
+3. Local automation is implemented on the top of the [Cake tool](https://cakebuild.net/). Check [**build/build.cake**](build/build.cake) if `Sitecore/Parameters.InitParams` are correct for your installation.
```
Comment (larionovadaria) [inline readme.md]: Looks like we have to move this folder back to src

File: readme.md [hunk 2]
```diff
@@ -115,43 +114,33 @@ To bootstrap the Commerce Server follow these instructions:
...
-14. In sitecore content editor modify **sitecore/Commerce/Catalog Management/Catalogs** item. Select **Habitat_Master** in the **Selected Catalogs** field.
-15. Publish content tree & Rebuild all the indexes in sitecore indexing manager.
+   * 17: change to **msBuildToolVersion: MSBuildToolVersion.VS2017**;
+   * 19: change to **msBuildToolVersion: MSBuildToolVersion.VS2019**.
+5. Change default IP address to VM address (if it's different) in
+- Wooli\src\publishsettings.targets: `192.168.50.4 -> VM IP`;
+- Wooli\build\build.cake: `192.168.50.4 -> VM IP`.
+6. Create a symbol link with `unicorn-wooli` name inside the **Root_Sitecore_Folder\App_Data** folder to the .\src folder (Root_Sitecore_Folded is the folder where Sitecore is installed, for ex. c:\inetpub\wwwroot\xp0.sc).
+7. In IIS bind **wooli.local** to the site, add **wooli.local** entry for localhost to the hosts list (C:\Windows\System32\drivers\etc\hosts), Wooli - is internal name of the Headless Commerce Accelerator.
+8. Run `.\build\build.ps1` in PowerShell with administration privileges (script will restore **NuGet** packages, install **npm** dependencies and run deployment process).
```
Comment (larionovadaria) [inline readme.md]: And here

File: readme.md [hunk 3]
```diff
@@ -115,43 +114,33 @@ To bootstrap the Commerce Server follow these instructions:
...
-
-10. Run "Default" task in Visual Studio Task Runner Explorer. 
-11. Create a symbol link with `unicorn-wooli` name inside the **Root_Sitecore_Folder\App_Data** folder to the .\src folder (Root_Sitecore_Folded is the folder where Sitecore is installed, for ex. c:\inetpub\wwwroot\xp0.sc)
-12. In IIS bind **wooli.local** to the site, add **wooli.local** entry for localhost to the hosts list (C:\Windows\System32\drivers\etc\hosts), Wooli - is internal name of the Headless Commerce Accelerator
-13. Log in to the sitecore then make a GET request on **http://{website}/unicorn.aspx?verb=Sync&log=null&skipTransparentConfigs=false**
-14. In sitecore content editor modify **sitecore/Commerce/Catalog Management/Catalogs** item. Select **Habitat_Master** in the **Selected Catalogs** field.
-15. Publish content tree & Rebuild all the indexes in sitecore indexing manager.
+   * 17: change to **msBuildToolVersion: MSBuildToolVersion.VS2017**;
+   * 19: change to **msBuildToolVersion: MSBuildToolVersion.VS2019**.
+5. Change default IP address to VM address (if it's different) in
```
Comment (larionovadaria) [inline readme.md]: Check again whether we need p5

File: readme.md [hunk 4]
```diff
@@ -161,3 +150,17 @@ In case of issues with the installation process you have several diagnostic opti
...
+Front-end code is processed using webpack. In order to start development server run following commands.
+For more details see [readme.md](src/readme.md) in the 'src' folder.
+```
+cd ./src
+npm start
+```
+
+----
+## Known issues
+1. If you have errors in Unicorn sync after run `.\build\build.ps1` you should log in to the sitecore then make a GET request on **http://{website}/unicorn.aspx?verb=Sync&log=null&skipTransparentConfigs=false**
```
Comment (larionovadaria) [inline readme.md]: And here

File: src/readme.md [hunk 5]
```diff
@@ -4,17 +4,22 @@
...
+| Command | Description |
+| --- | --- |
+| `start` | Starts local development server with configuration for project **Wooli** and env **local**. Configuration information stores in [manifest.json](Project/Wooli/client/manifest.json) file |
+| `build:Debug` | Builds client code in Debug mode |
+| `sc:codegen` | Starts *.ts and *.cs models generation for sitecore templates |
+| `test` | Runs client tests |
+| `test-watch` | Starts watching for new client tests and reruns them automatically |
+| `test-cover` | Runs client tests with generation of coverage results |
 
- `npm start -- --project=Wooli --env=local`, command runs local dev server and fetches a sitecore context from a url specified in **manifest** file of a project
```
Comment (larionovadaria) [inline src/readme.md]: Why was this one removed?

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/readme.md (hunk 5 above)
Comment (yuya-yuliya) [reply src/readme.md]: We move parameters inside command specification `"start": "node ./scripts/webpack/environments/development/server.js --project=Wooli --env=local"` to simplify running process

File: readme.md [hunk 6]
```diff
@@ -115,49 +114,53 @@ To bootstrap the Commerce Server follow these instructions:
...
 * Make sure the following exists in your web.config and if not then add it (system.webServer/handlers section):
 `<add verb="*" path="sitecorejss_media.ashx" type="Sitecore.JavaScriptServices.Media.MediaRequestHandler, Sitecore.JavaScriptServices.Media" name="Sitecore.JavaScriptServices.Media.MediaRequestHandler" />`
 
 ----
 ### Building & deploying Headless Commerce Accelerator      
-1. Fetch **Headless Commerce Accelerator** code base
+1. Fetch **Headless Commerce Accelerator** code base.
 2. Copy Sitecore license to **./src** folder.
-3. Local automation is implemented on the top of the [Cake tool](https://cakebuild.net/). Check **build/build.cake** if `Sitecore/Parameters.InitParams` are correct for your installation.
+3. Local automation is implemented on the top of the [Cake tool](https://cakebuild.net/). Check [**build/build.cake**](src/build.cake) if `Sitecore/Parameters.InitParams` are correct for your installation.
```
Comment (larionovadaria) [inline readme.md]: [**build/build.cake**](src/build.cake) ==> [**src/build.cake**](src/build.cake)

**Review by ghost — APPROVED** [review]

**Pull request merged**